tests/std/tests/GH_004477_mdspan_warning_5246/test.compile.pass.cpp

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <array>
+#include <mdspan>
+
+using namespace std;
+
+// Test GH-4477 "Using std::mdspan results in multiple warnings (C5246)"
+void test_gh_4477() {
+    array test_data{0, 1, 2, 3, 4, 5};
+    // mdspan(329,54): warning C5246 + mdspan(124,50): warning C5246

```

Comment:
This comment isn't providing any value - the test name and bug title have already mentioned `mdspan` and C5246, and the line/column numbers will become outdated.